ccw.core.test/src/clj/ccw/util_test.clj

```diff
@@ -0,0 +1,59 @@
+(ns ccw.util_test
+  (:use clojure.test
+        ccw.util))
+
+(defmacro with-private-fns [[ns fns] & tests]
+  "Refers private fns from ns and runs tests in context."
```

Comment:
The docstring is not in the right position, it should come after `with-private-fns`